dynawo/sources/Modeler/DataInterface/IIDM/DYNVoltageLevelInterfaceIIDM.cpp

```diff
@@ -415,4 +415,10 @@ VoltageLevelInterfaceIIDM::mapConnections() {
   }
 }
 
+std::vector<std::string>
+VoltageLevelInterfaceIIDM::getBusesConnectedBySwitch(const std::string&) {
+  // this service is not supported by IIDM 1.0
```

Comment:
I'd add a dynawo log notifying the user he can not use it for iidm 1.0

Reply:
DEBUG ? If it is warning or error, I'm not sure since it is a feature for further version

Reply:
should be a throw, and if we change the architecture on this it won't be needed anymore

Reply:
(there is not future for the IIDM folder)

Reply:
This case cannot happen in IIDM1.0 so we consider that we are always in the case "2 buses cannot be linked by switches" in this case. Comment will be updated